src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -7,5 +7,6 @@ public enum DocumentFlags
     {
         None = 0,
         Versioned = 1,
+        Artificial = 2,
```

Comment:
We can always change this as long that the value will remain 2. We may reuse the artificial documents for other features.